tests/functional_tests/L2_NeMo_2_Conversion_Test_Baichuan2.sh

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020-2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+HF_ORI_PATH=/home/TestData/llm/models/baichuan2_toy
```

Comment:
Hey Ao. These `tests/functional_tests/L2_NeMo_2_Conversion_Test_*.sh` scripts differ only by `HF_ORI_PATH`, `NEMO_MODEL_TYPE`, etc. variables, right?

Suggestion: maybe we can have a single `L2_NeMo_2_Conversion_Test.sh` script and export the variables in the main `cicd-main.yml` file? This could help adding new models and also applying any changes to `test_import_from_hf.py` / `test_export_to_hf.py` scripts later.